ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -317,6 +311,44 @@ func convertToContainerDef(inputCfg *adapter.ContainerConfig, ecsContainerDef *C
 	// Set fields from ecs-params file
 	if ecsContainerDef != nil {
 		outputContDef.Essential = aws.Bool(ecsContainerDef.Essential)
+
+		overrideMsg := "Using ecs-params value as override (was %v but is now %v)"
+		if ecsContainerDef.Cpu > 0 {
+			if outputContDef.Cpu != nil && *outputContDef.Cpu > 0 {
+				log.WithFields(log.Fields{
```

Comment:
I'm wondering if it'd be possible to DRY up the logging code since it's repeated for all the fields. I like how you have the template string that's reused from line 315, so perhaps it could be incorporated with the logging into a helper function?